react-native/react/native/remote-manager.desktop.js

```diff
@@ -15,19 +16,26 @@ import type {Action, Dispatch} from '../constants/types/flux'
 
 import type {TrackerState} from '../reducers/tracker'
 import type {PinentryState} from '../reducers/pinentry'
+import type {ShowUpdateState} from '../reducers/update'
 
 export type RemoteManagerProps = {
   registerPinentryListener: () => void,
+  registerUpdateListener: () => void,
   pinentryOnCancel: (sessionID: number) => void,
   pinentryOnSubmit: (sessionID: number, passphrase: string, features: GUIEntryFeatures) => void,
   registerIdentifyUi: () => void,
   registerTrackerChangeListener: () => void,
   onCloseFromHeader: () => void,
   trackerServerStarted: boolean,
-  startTimer: (dispatch: Dispatch, getState: any) => void,
-  stopTimer: () => Action,
+  trackerStartTimer: (dispatch: Dispatch, getState: any) => void,
+  trackerStopTimer: () => Action,
   trackers: {[key: string]: TrackerState},
-  pinentryStates: {[key: string]: PinentryState}
+  pinentryStates: {[key: string]: PinentryState},
+  showUpdateState: ShowUpdateState,
+  updateOnSkip: () => void,
```

Comment:
that's one option. it hink it'd be better if this thing knew about components and what remote windows they need (maybe not even that) but i don't think it needs to know about all the props they get